src/VirtoCommerce.ElasticSearchModule.Data/ElasticSearchRequestBuilder.cs

```diff
@@ -9,16 +9,21 @@ namespace VirtoCommerce.ElasticSearchModule.Data
 {
     public class ElasticSearchRequestBuilder
     {
+        
+        // Used to map 'score' sort field to Elastic Search _score sorting field 
+        private const string Score = "score";
         public virtual ISearchRequest BuildRequest(SearchRequest request, string indexName, Properties<IProperties> availableFields)
         {
+            
             var result = new Nest.SearchRequest(indexName)
             {
                 Query = GetQuery(request),
                 PostFilter = GetFilters(request, availableFields),
                 Aggregations = GetAggregations(request, availableFields),
                 Sort = GetSorting(request?.Sorting),
                 From = request?.Skip,
-                Size = request?.Take
+                Size = request?.Take,
+                TrackScores = request?.Sorting.Any(x=>x.FieldName == Score) ?? false
```

Comment:
Could you use our extension `string.EqualsInvariant` to compare strings

Reply:
Fixed